server/index.js

```diff
@@ -1,12 +1,113 @@
 const express = require('express');
+const axios = require('axios');
+const cors = require('cors');
 const app = express();
 const port = 3000;
 
 app.use(express.json());
+app.use(cors());
 app.use(express.urlencoded({ extended: false }));
 app.use('/items/:itemId', express.static('client'));
 
 app.listen(port, () => {
 	// eslint-disable-next-line no-console
 	console.log(`Listening at http://localhost:${port}`);
 });
+
+app.get('/serviceBundles', (req, res) => {
+	let images = axios.get('http://localhost:3006/items/1/bundle.js');
```

Comment:
Create a holder object which is say {images: "localhost:3006"} with a variable from a config file